### PR TITLE
HUB #9: Actually HUB#3, but why change horses in the middle of a race

### PR DESCRIPTION
**Pull request by kbrimm**
Hub connects to back end endpoint, transmits generated data using JSON object and requests data.

### CODE REVIEW

**Comment by SoySauceLabs** [comment]
Woooooppps, forgot that with the addition of the Hub repo comes a "Hub Board", which I didn't check as "visible". _Now_ I see this PR. I'll look at it today. 👼

File: README.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
-# plantalytics-hub
+# Plantalytics Conduit
+
 Scripts to enable a MultiConnect Conduit to act as a hub for the Plantalytics system data collection nodes.
 
+## Setup
+
+Setting up the Conduit is 9/10ths of the battle. The first step is to configure the network settings.
```
Comment (SoySauceLabs) [inline README.md:7]: GGGGGGG IIIIIII JOOOOOEEEE!!!!

File: data_generator.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+def load_config():
+    config = ConfigParser.RawConfigParser()
+    config.read('hub.config')
+    # Load hub data
+    global HUB_ID 
+    HUB_ID = config.get('Hub Data', 'hub_id')
+    global VINE_ID
+    VINE_ID = config.get('Hub Data', 'vine_id')
+    global PUBLIC_KEY
+    PUBLIC_KEY = config.get('Hub Data', 'public_key')
```
Comment (SoySauceLabs) [inline data_generator.py:25]: Heads up for the key: I've modified the endpoint to actually check this key for "auth". The key it's currently expecting is in our `env_var.txt file` in the doc repo. You'll want to make sure to populate `hub.config` with that value.

File: data_generator.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    object['hub_data'] = []
+    # For each node, create a data set and add it to the object.
+    for n in range(0, itemCount):
+        object['hub_data'].append(get_node_data(n))
+    # Add the batch timestamp.
+    object['batch_sent'] = int(time.time()*1000)
+    print object
+    return object
+
+def bedtime():
```
Comment (SoySauceLabs) [inline data_generator.py:69]: Feeling sleeeeeeepy 😴

**Comment by SoySauceLabs** [comment]
I'm _feeling_ it!
...
__SIP IT!__ 🍷

File: data_generator.py (hunk 2 above)
Comment (kbrimm) [inline data_generator.py:25]: I will do this thing.

**Comment by kbrimm** [comment]
Is this good to go, then?

**Pull request merged**